test/jasmine/tests/violin_test.js

```diff
@@ -533,7 +550,7 @@ describe('Test violin hover:', function() {
 
             Plotly.plot(gd, fig).then(function() {
                 mouseEvent('mousemove', 300, 250);
-                assertViolinHoverLine([299.35, 250, 250, 250]);
+                assertViolinHoverLine([178.67823028564453, 250, 80, 250]);
```

Comment:
@Kully do you _why_ this test had to be modified?

Reply:
I wasn't too sure to be honest. I ran the test a few times but there were only two values the assert was saying was valid, iirc.

Reply:
Presumably it changed because autorange can shrink depending on `side` now, so the violin moved.